modules/ROOT/pages/apikit-4-for-soap.adoc

```diff
@@ -1,21 +1,18 @@
-= Workflow: Implement a SOAP Interface
+= APIkit for SOAP
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: apikit, soap
 
-In this workflow, you create an API using Studio and APIkit for SOAP. APIkit for SOAP supports consuming a remote WSDL. This workflow, however, uses a local WSDL that you download. The SOAP interface you create accesses data resources and integrates the interface with backend processes that function as an API.
+APIkit for SOAP is a framework designed to simplify the development and integration of SOAP-based APIs. SOAP (Simple Object Access Protocol) is a messaging protocol used for exchanging structured information between web services. With APIkit for SOAP, you get a set of tools and features that streamline SOAP API development, ensuring seamless integration with your existing SOAP infrastructure. APIkit for SOAP simplifies the handling of SOAP message formats and headers, so you can focus on the core logic of your APIs. Additionally, it automates the binding to WSDL (Web Services Description Language), reducing manual configuration efforts and ensuring accuracy.
 
-. xref:apikit-4-soap-prerequisites-task.adoc[Meet prerequisites for using APIkit for SOAP]
-. xref:apikit-4-soap-project-task.adoc[Create an APIkit for SOAP project]
-. xref:apikit-4-soap-fault-task.adoc[Identify Operations to Implement]
-. xref:apikit-4-prevent-fault-task.adoc[Prevent the SOAP fault]
-. xref:apikit-4-get-header-task.adoc[To get a SOAP header]
-. xref:apikit-4-set-header-task.adoc[To set a SOAP header]
-. xref:apikit-4-display-fault-task.adoc[To display a typed fault]
+== Features
 
+* Automatic binding to you WSDL definitions
+* Efficient handling of SOAP messages
+* Support for data transformations
+
+By placing emphasis on SOAP/WSDL contracts, with APIkit for SOAP you define and expose SOAP APIs with ease. Its capabilities in managing SOAP message formats and headers ensure seamless integration with SOAP infrastructure. With APIkit for SOAP, you can create robust and reliable SOAP-based APIs, accelerating your development workflow and delivering enhanced integration capabilities.
```

Comment:
```suggestion
By emphasizing SOAP/WSDL contracts, with APIkit for SOAP, you define and expose SOAP APIs with ease. Its capabilities in managing SOAP message formats and headers ensure seamless integration with SOAP infrastructure. With APIkit for SOAP, you can create robust and reliable SOAP-based APIs, accelerating your development workflow and delivering enhanced integration capabilities.
```

Reply:
This seems more intro - any reason it's not included there, rather than in Features?

Reply:
I tried to make a closing paragraph.
No problem, I'll include this information in the intro and/or in the features.
Thanks!